yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1235,7 +1237,7 @@ func fetchUnsupportedPlPgSQLObjects(schemaAnalysisReport utils.SchemaReport) []U
 			})
 		}
 		feature := UnsupportedFeature{
-			FeatureName: reason,
+			FeatureName: issueName,
```

Comment:
This will have callhome/yugabyted implications, no? Do you want to keep it the same now (set it to reason), and we can discuss this later when dealing with callhome/yugabyted?

Reply:
The issue with grouping by reason(/description) anymore is that it is dynamic(`%s`s), having column info etc..

There will be new unnecessary groups if we go that route for different value of %s in each case.